### PR TITLE
Contribute blues wireless driver to repo

### PR DESCRIPTION
**Pull request by bytewizer**
## Checklist:

- [x] I've placed my contribution in the appropriate section.
- [x] I've choosen a good name for it and I've checked the repo for duplicates.
- [x] I've added a readme.md following the provided template (TBD). 
- [x] My code follows the code style of this project.
- [x] I've tidy my files inside the contribution folder.
- [x] I've edited the repo README.md to include the mention and link to my contribution. It's under the proper section and the link is correct.
- [x] The code builds.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/nf-Community-Contributions?pullRequest=42) <br/>All CLA requirements met.

File: drivers/Blues-Wireless/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Blues Wireless Notecarrier-AF with Adafruit Feather ESP32
+
+The Notecard combines prepaid cellular connectivity and "off-the-internet" communications connecting any device to cloud services like AWS and Azure. 
+
+![Notecard](https://raw.githubusercontent.com/bytewizer/blueswireless/develop/images/notecard-esp32.jpg)
+
+## .NET nanoFramework Driver Libraries
+
+The folloiwing repository contains libraries for working with [Blues Wireless Notecard](https://blues.io) and [.NET nanoFramework](https://www.nanoFramework.net/).
+* <a href="https://github.com/bytewizer/blueswireless/tree/develop/src/Bytewizer.NanoCLR.Drivers.Blues.Notecard">Notecard Driver</a>
```
Comment (Ellerbach) [inline drivers/Blues-Wireless/README.md]: please use the Markdown way of making links :-) 
```suggestion
* [Notecard Driver](https://github.com/bytewizer/blueswireless/tree/develop/src/Bytewizer.NanoCLR.Drivers.Blues.Notecar)
```

**Review by Ellerbach — APPROVED** [review]
Please adjust the readme with the suggestion and we will be good to go. Thanks!

**Comment by bytewizer** [comment]
Adjustments completed

**Pull request merged**